### PR TITLE
Fix/#57 remove useless plugin attributes

### PR DESCRIPTION
**Pull request by raschju**
## Summary

Attribute windowLength does not exist and therefore has been removed.

### CODE REVIEW

File: packages/clients/afm/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## 1.0.1
 
 - Fix: The included example files have been updated to the new syntax and work again.
+- Fix: Remove configurable attribute windowLength which does not exist anymore.
```
Comment (dopenguin) [inline packages/clients/afm/CHANGELOG.md:6]: Changelog entry is not necessary, as the respective customer does not recognize that parameter.

File: packages/clients/dish/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 1.0.1
+
+- Fix: Remove configurable attribute windowLength which does not exist anymore.
+
```
Comment (dopenguin) [inline packages/clients/dish/CHANGELOG.md:6]: Changelog entry is not necessary, as the respective customer does not recognize that parameter.

**Review by dopenguin — CHANGES_REQUESTED** [review]
🏓 @raschju @KlampMe

File: packages/clients/dish/CHANGELOG.md (hunk 2 above)
Comment (KlampMe) [reply packages/clients/dish/CHANGELOG.md]: thanks for the note 🏓

**Review by dopenguin — APPROVED** [review]
🚀 @KlampMe @raschju

**Pull request merged**